packages/client/src/hoc/withChecker.tsx

```diff
@@ -44,7 +45,7 @@ export default (Component: any) => (props: Props) => {
 
   const initChecker = async () => {
     // if the topic is not found (dynamic IMTR-checker) or the topic is found and has an imtr flow
-    if (!checker && !error && (!topic || topic.hasIMTR)) {
+    if (!checker && !error && topic && topic.hasIMTR) {
```

Comment:
We need to also follow this path in case we don't have a topic, for dynamic imtr-checker.